### ISSUE TITLE
parameter '_stacklevel' of torch.nn.functional.{softmax|log_softmax} can set to string

### ISSUE THREAD

**Issue by cheyennee** [pytorch/pytorch#83164]
### 🐛 Describe the bug

I'm not sure whether the parameter '_stacklevel' of torch.nn.functional.{softmax|log_softmax} can set to string since the documentation doesn't explain what _stacklevel is, why it's set to 3, what's the type of it and what to do with it.

```
import torch
results={}
arg_1 = torch.rand([4, 12, 141, 141], dtype=torch.float32)
arg_2 = -1
arg_3 = "max"
results['res'] = torch.nn.functional.softmax(arg_1,arg_2,_stacklevel=arg_3,)
```

```
import torch
results={}
arg_1 = torch.rand([5, 4, 8, 48], dtype=torch.float32)
arg_2 = 1
arg_3 = "max"
results['res'] = torch.nn.functional.log_softmax(arg_1,arg_2,_stacklevel=arg_3,)
```

### Versions

pytorch: 1.8.1
python: 3.8.3
os: win11

**Comment by cheyennee** [comment]
Same problem can be found in torch.nn.functional.softmin.
```
import torch
results={}
arg_1 = torch.rand([2, 3], dtype=torch.float32)
arg_2 = 0
arg_3 = 'max'
results['res'] = torch.nn.functional.softmin(arg_1,arg_2,_stacklevel=arg_3,)
```
Above code works

**Comment by cheyennee** [comment]
From vadimkantorov's response:
I think the essence of the story is that _stacklevel is passed to some warnings.warn(...) function or even is unused now / cast implicitly. So it's not very interesting to inquire about its possible values.

So this parameter can be set to any type.
